csrc/ed_gen.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+#include "auto_free.h"
+#include "bn.h"
+#include "buffer.h"
+#include "env.h"
+#include "generated-headers.h"
+#include "keyutils.h"
+#include "util.h"
+#include <openssl/bn.h>
+#include <openssl/evp.h>
+#include <memory>
```

Comment:
Clean up your includes. You seem to have more includes that actual symbols here which suggests you're including more than you use. Also, follow the convention of system libraries up top and all project headers below.

The Google [C-style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) is excellent if you want to improve your style.

Reply:
Updated